common/pluginmanager.py

```diff
@@ -91,14 +91,19 @@ def error(self, code, message):
 
                 Known error codes:
 
-                1. No or no valid configuration
+                1: No or no valid configuration
```

Comment:
I assume that new codes should be mentioned in the [user-callback](https://github.com/bit-team/user-callback) repo also.

Reply:
Right, I forgot to update the README.md there!

I will update this too today

Reply:
Done